jsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["*"]
+    }
+  }
+}
```

Comment:
If Typescript is supported this file could be eliminated in favor of the `tsconfig.json`